pkg/controller/networkpolicy/networkpolicy.go

```diff
@@ -66,7 +67,17 @@ func NetworkPolicyForApp(req router.Request, resp router.Response) error {
 			Ingress: []networkingv1.NetworkPolicyIngressRule{{
 				From: allowedNamespaceSelectors,
 			}},
-			PolicyTypes: []networkingv1.PolicyType{networkingv1.PolicyTypeIngress},
+			Egress: []networkingv1.NetworkPolicyEgressRule{{
+				To: []networkingv1.NetworkPolicyPeer{{
+					IPBlock: &networkingv1.IPBlock{
+						CIDR: "0.0.0.0/0",
+						Except: []string{
+							"169.254.169.254/32",
```

Comment:
What is this address? It will definitely seem like a magic number in the future so can we make this a constant with a name that explains what it is?

Reply:
It's a local IP address that is AWS-specific for fetching EC2 instance metadata:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

Yeah that's a good call. I'll define it as a constant.

Reply:
Nice! Thanks.